Userland/Libraries/LibWeb/CSS/Properties.json

```
@@ -960,6 +960,15 @@
     ],
     "percentages-resolve-to": "length"
   },
+  "column-span": {
+    "animation-type": "discrete",
+    "inherited": false,
+    "initial": "none",
+    "valid-identifiers": [
+      "none",
+      "all"
+    ]
```

This isn't obvious (I should probably document it somewhere) but if you have an entry in `Enums.json` for this property, you can instead do this rather than duplicating the list of identifiers:

```suggestion
    "valid-types": [
      "column-span"
    ]
```

---

Ah, that's good to know. It took me a while to even figure out that `Properties.json` was generating code 😅 and I was editing the generated code.

Fixed!